torch_xla_py/xla_dist.py

```diff
@@ -21,16 +21,35 @@
 class Worker(object):
 
   def __init__(self, internal_ip, machine_type, zone):
-    self._internal_ip = internal_ip
-    self._machine_type = machine_type
-    self._zone = zone
+    self._internal_ip = str(internal_ip)
+    self._machine_type = str(machine_type)
+    self._zone = str(zone)
```

Comment:
why are these str?

Reply:
I added them to be string to be consistent to consider different types. More significant for stuff like `port` where I realized that `"80"` != `80`.

Reply:
They should not be needed. They should all be string in input, and if not, it is call site error.

Reply:
Done.